### PR TITLE
Added the ability to record when a recipe was made

### PR DESCRIPTION
**Pull request by boblail**
Here's what it looks like when you click **I made this**:

screen shot 2017-08-20 at 4 51 34 pm

(I'm sure that'll get so more designing as we go...)

And this lets you filter your recipes to show only experimental or familiar ones:

screen shot 2017-08-20 at 4 51 49 pm

### CODE REVIEW

File: app/helpers/date_helper.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module DateHelper
+
+  def days_ago_in_words(date)
+    distance_in_days = Date.today - date
```
Comment (clettenberg) [inline app/helpers/date_helper.rb]: @boblail This returns a fraction (4/1) or (10/1). Just need to wrap it in a `.to_int`

```ruby
distance_in_days = (Date.today - date).to_i
```

File: db/schema.rb [hunk 2]
```diff
@@ -41,6 +41,17 @@
     t.datetime "updated_at"
   end
 
+  create_table "preparations", force: :cascade do |t|
+    t.bigint "recipe_id", null: false
```
Comment (clettenberg) [inline db/schema.rb]: Just curious, why `bigint` instead of `integer`?

File: app/helpers/date_helper.rb (hunk 1 above)
Comment (boblail) [inline app/helpers/date_helper.rb]: Good call! I made that fix and amended the commit.

File: db/schema.rb (hunk 2 above)
Comment (boblail) [inline db/schema.rb]: I don't know!

I wrote:

```ruby
t.belongs_to :recipe, null: false, foreign_key: { on_delete: :cascade }
```

in the migration.

I wonder if some version of Rails changed the default... ?

Comment (clettenberg) [inline db/schema.rb]: Interesting! You're guess is as good as mine ¯\_(ツ)_/¯

**Pull request merged**